### DISCUSSION TITLE
from元素InputNumber框长度很短

### DISCUSSION THREAD

**Discussion by JokieJae** [vbenjs/vue-vben-admin#1713] [Q&A]
![image](https://user-images.githubusercontent.com/91403595/158091191-25786de6-74d1-4321-a509-f8e0dcbc0904.png)

**Comment by Chaos-Enyo** [comment]
` {
      field: '',
      label: '',
      component: 'InputNumber',
      componentProps: {
        style: { width: '100%' },
      },
    },`
这么写就好了。
